crates/symbolicator-service/src/caching/metadata.rs

```diff
@@ -17,15 +17,47 @@ pub struct Metadata {
     /// This is serialized with second precision.
     #[serde(with = "timestamp")]
     pub time_created: SystemTime,
+
+    /// Optional data for debugging.
+    ///
+    /// See [Self::from_key] for how this is populated.
+    pub debug: Option<Arc<str>>,
```

Comment:
I think we already should make this an object, an object can be easily extended but switching from a string to object is a breaking change (or needs a special deserializer).